modules/cluster-logging-updating-logging-to-46.adoc

```diff
@@ -0,0 +1,321 @@
+
+.. Click *Install*.
+
+.. Wait for a few seconds, then click *Operators* -> *Installed Operators*.
++
+The Elasticsearch Operator is shown as 4.6. For example:
++
+[source,terminal]
+----
+Elasticsearch Operator
+4.6.0-202007012112.p0 provided
+by Red Hat, Inc
+----
++
+Wait for the *Status* field to report *Succeeded*.
+
+. Update the Cluster Logging Operator:
+
+.. From the OpenShift web console, click *Operators* -> *Installed Operators*.
+
+.. Select the `openshift-logging` project.
+
+.. Click the *Cluster Logging*.
+
+.. Click *Subscription* -> *Channel*.
+
+.. In the *Change Subscription Update Channel* window, select *4.6*.
+
+.. On the *YAML* tab, set the retention policy in the `clusterlogging` custom resource (CR). You must specify the length of time that Elasticsearch should retain each log source. Enter an integer for the number of days, for example, `7d` for seven days. The `application` `maxAge` can be less than or equal to `7d`. The `infra` `maxAge` and `audit` `maxAge` must be `0d`, and you must *explicitly set this value* to override the default. Logs older than the `maxAge` are deleted. You must specify a retention policy for each log source or the Elasticsearch indices will not be created for that source.
++
+[source,terminal]
+----
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
+spec:
+  managementState: "Managed"
+  logStore:
+    type: "elasticsearch"
+    retentionPolicy:
+      application:
+        maxAge: 7d
+      infra:
+        maxAge: 0d
+      audit:
+        maxAge: 0d
+...
+----
+
+.. Click *Save*.
+
+.. Wait for a few seconds, then click *Operators* -> *Installed Operators*.
++
+The Cluster Logging Operator is shown as 4.6. For example:
++
+[source,terminal]
+----
+Cluster Logging
+4.6.0-202007012112.p0 provided
+by Red Hat, Inc
+----
++
+Wait for the *Status* field to report *Succeeded*.
+
+. Check the logging components:
+
+.. Ensure that all Elasticsearch pods are in the *Ready* status:
++
+[source,terminal]
+----
+$ oc get pod -n openshift-logging --selector component=elasticsearch
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                            READY   STATUS    RESTARTS   AGE
+elasticsearch-cdm-1pbrl44l-1-55b7546f4c-mshhk   2/2     Running   0          31m
+elasticsearch-cdm-1pbrl44l-2-5c6d87589f-gx5hk   2/2     Running   0          30m
+elasticsearch-cdm-1pbrl44l-3-88df5d47-m45jc     2/2     Running   0          29m
+----
++
+.. Ensure that the Elasticsearch cluster is healthy:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch elasticsearch-cdm-1pbrl44l-1-55b7546f4c-mshhk -- es_cluster_health
+----
++
+[source,json]
+----
+{
+  "cluster_name" : "elasticsearch",
+  "status" : "green",
+}
+....
+
+----
+
+.. Ensure that the Elasticsearch cron jobs are created:
++
+[source,terminal]
+----
+$ oc project openshift-logging
+----
++
+[source,terminal]
+----
+$ oc get cron job elasticsearch-*
+----
++
```

Comment:
Perhaps label this code block as .Example output

Reply:
Done.